pages/npcs.html

```diff
@@ -4,166 +4,200 @@
-            <header>
-                <img src="/images/TempLogo.jpg" alt="Temporary profile pic" height="50" width="50">
-                <h4>Elemental Locus of Caustic Waters</h4>
-                <h5>Water Elemental</h5>
+            <header class="clearfix">
+                <img src="/images/TempLogo.jpg" alt="Temporary profile pic" class="profilepic">
+                <h3>Elemental Locus of Caustic Waters</h3>
+                <hr class="header2">
+                <h4>Water Elemental</h4>
             </header>
             <section>
-                <p>
+                <p class="chardesc">
                    Formed in the amber bones of a massive coelacanth, the Elemental Locus of Caustic Waters sat on top of a secondary cutting of Knot's home tree. Upon defeat, the cutting known as the Primal Wellspring was reawakened.
                 </p>
                <h6>Defeated.</h6>
             </section>
         </article>
        </div>
+    </div>
        <!--Begin Left Monster Profile Div 3-->
-       <div>
+       <div class="twocolumn">
+       <div class="leftcolumnalt2">
         <article id="fire">
-            <header>
-                <img src="/images/TempLogo.jpg" alt="Temporary profile pic" height="50" width="50">
-                <h4>Elemental Locus of Coruscating Flame</h4>
-                <h5>Fire Elemental</h5>
+            <header class="clearfix">
+                <img src="/images/TempLogo.jpg" alt="Temporary profile pic" class="profilepic">
+                <h3>Elemental Locus of Coruscating Flame</h3>
+                <hr class="header2">
+                <h4>Fire Elemental</h4>
             </header>
             <section>
-                <p>
+                <p class="chardesc">
                    The Elemental Locus of Coruscating Flame was an elemental that had coalesced from the iron bones of a Primal wolf which sat on top of a secondary cutting of Knot's home tree. Upon defeat, the  cutting known as the Primal Forge was reawakened.
                 </p>
                 <h6>Defeated.</h6>
             </section>
         </article>
        </div>
        <!--Begin Right Monster Profile Div 4-->
-       <div>
+       <div class="rightcolumnalt2">
         <article id="air">
-            <header>
-                <img src="/images/TempLogo.jpg" alt="Temporary profile pic" height="50" width="50">
-                <h4>Elemental Locus of Battering Winds</h4>
-                <h5>Air Elemental</h5>
+            <header class="clearfix">
+                <img src="/images/TempLogo.jpg" alt="Temporary profile pic" class="profilepic">
+                <h3>Elemental Locus of Battering Winds</h3>
+                <hr class="header2">
+                <h4>Air Elemental</h4>
             </header>
             <section>
-                <p>
+                <p class="chardesc">
                    An air elemental formed on the bones of a Primal Pteradon.
                 </p>
                <h6>Defeated.</h6>
```

Comment:
think of the headings and heading levels like a table of contents. i recommend that you don't use `h5` or `h6` on these profiles and instead use a `span` tag with a class to style it as desired.

Reply:
@axamei Got it! I have just made the updates.